src/content/self-hosted/install/backup-restore.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Backup and Restore
+description: Backup and Restore your Okteto instance
+id: backup
+sidebar_label: Backup and Restore
+---
+
+Backup and restore planning is essential to ensure the availability and integrity of your Okteto instance in case of unexpected failures or disaster recovery.
+This guide outlines the steps to recover an Okteto instance in such scenarios.
+
+## Critical Resources
+
+To effectively recover an Okteto instance, focus on the following critical resources:
+
+- **Service Accounts**: Preserve service accounts associated with users. These service accounts reside within the `okteto` namespace and are labeled with `dev.okteto.com: "true"`.
+- **Secrets**: Recreate all secrets within the `okteto` namespace.
```

Comment:
```suggestion
- **Secrets**: Recreate all secrets within the `okteto` namespace.
```

The first sentence in this section says these are the most critical resources to "focus on" to effectively recovery an Okteto instance.

The first resource (Service Accounts) seems to be worded in a way that suggests "you should _backup_ any service accounts linked to users".

Now this resource (Secrets) seems to describe what the outcome would be of restoring Secrets, which deviates from the previous resource.

Is the intent here to say: "You should _backup_ Secrets because it will save you from having to recreate all of them when restoring the instance"?

Reply:
You are right, I will update based on your suggestion